scala/skeleton/build.sbt

```diff
@@ -108,19 +108,19 @@ libraryDependencies ++= Seq(
   "com.ibm.icu"                       % "icu4j"                            % "55.1",
   "com.typesafe"                      % "config"                           % "1.3.0",
   "com.typesafe.scala-logging"       %% "scala-logging"                    % "3.1.0",
-  "com.typesafe.slick"               %% "slick"                            % "3.0.0",
+  "com.typesafe.slick"               %% "slick"                            % "3.0.2",
   "com.univocity"                     % "univocity-parsers"                % "1.5.6",
-  "org.apache.commons"                % "commons-compress"                 % "1.9",
+  "org.apache.commons"                % "commons-compress"                 % "1.10",
   "org.apache.commons"                % "commons-lang3"                    % "3.4",
   "org.apache.commons"                % "commons-math3"                    % "3.5",
   "org.apache.httpcomponents"         % "httpclient"                       % "4.5",
   "org.joda"                          % "joda-money"                       % "0.10.0",
-  "org.jsoup"                         % "jsoup"                            % "1.8.2",
+  "org.jsoup"                         % "jsoup"                            % "1.8.3",
   "org.scalactic"                    %% "scalactic"                        % "2.2.5",
   "org.mockito"                       % "mockito-core"                     % "1.10.19"      % Test,
   "org.scalamock"                    %% "scalamock-scalatest-support"      % "3.2.2"        % Test,
   "org.scalatest"                    %% "scalatest"                        % "2.2.5"        % Test,
-  "org.seleniumhq.selenium"           % "selenium-java"                    % "2.46.0"       % Test
+  "org.seleniumhq.selenium"           % "selenium-java"                    % "2.47.1"       % Test
```

Comment:
in the future i don't think it's necessary for you to always have to update versions here, seems like a PITA